src/activeList.js

```diff
@@ -22,9 +43,41 @@ const handleItem = function (itemData) {
         // eslint-disable-next-line no-unneeded-ternary
         currentItem.completed = currentItem.completed ? false : true;
         itemsArr.splice(itemIndex, 1, currentItem);
-        // eslint-disable-next-line no-use-before-define
+        // eslint-disable-next-line eqeqeq
+        if (currentItem.completed == false) {
+          item.querySelector('.tast').style.textDecoration = '';
+          item.querySelector('.drag-drop').style.display = 'flex';
+          item.querySelector('.trash').style.display = 'none';
+        } else {
+          item.querySelector('.tast').style.textDecoration = 'line-through';
+          item.querySelector('.drag-drop').style.display = 'none';
+          item.querySelector('.trash').style.display = 'flex';
+        }
+        removeAll.addEventListener('click', (e) => {
+          e.preventDefault();
+          // eslint-disable-next-line eqeqeq
+          if (currentItem.completed == true) {
+            const removeIndex = itemsArr.indexOf(currentItem);
+            itemsArr.splice(removeIndex, 1);
```

Comment:
- [ ] You should have A **function** for the "Clear all completed" button using the `Array.filter()` method. Let's fix that